manifests/admin.pp

```diff
@@ -0,0 +1,17 @@
+#
+# == Class: pulp::admin
+#
+# Install and configure Pulp admin
+#
+class pulp::admin (
+  $version           = 'installed',
+  $host,
+  $port              = 443,
+  $api_prefix        = '/pulp/api',
+  $verify_ssl        = true,
+  $ca_path           = '/etc/pki/tls/certs/ca-bundle.crt',
+  $upload_chunk_size = '1048576',
+  $puppet            = false,) {
```

Comment:
We try to keep class parameters in the params.pp style, these should move to `pulp::admin::params`